web/simonamarcu2/css/my-cv-v3-starter/index.html

```diff
@@ -5,27 +5,41 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Simona's my cv v3 starter</title>
-    <link rel="stylesheet" href="my-cv-v3.css">
-    <meta name="description" content="My project cv learned in codetap academy,started in 2021. Mentored by Marian Zburlea and with the help of on Daily and Nigtly mentored team. Starting a new adventure in coding , makes me happy.">
+    <link rel="stylesheet" href="my-cv-v3.css" />
+    <meta
+      name="description"
+      content="My project cv learned in codetap academy,started in 2021. Mentored by Marian Zburlea and with the help of on Daily and Nigtly mentored team. Starting a new adventure in coding , makes me happy."
+    />
   </head>
   <body>
-    <div class="gridul-meu">
-      <div class="patrat">baby</div>
-      <div class="patrat">toddler</div>
-      <div class="patrat">kid</div>
-      <div class="patrat">adult</div>
-      <div plin class="dreptunghi"></div>
-      <div plin class="dreptunghi"></div>
-      <div plin class="dreptunghi"></div>
-      <div plin class="dreptunghi"></div>
-      <div class="dreptunghi"></div>
-      <div class="dreptunghi"></div>
-      <div class="dreptunghi"></div>
-      <div class="dreptunghi"></div>
-      <div class="dreptunghi"></div>
-      <div class="dreptunghi"></div>
-      <div class="dreptunghi"></div>
-      <div class="dreptunghi"></div>
+    <div data-full>
+      <h1><span>Simona</span> <span>Marcu</span></h1>
```

Comment:
Why is every word wrapped in a `span` element? Does it have the same effect if you write this line of code in this way: 
```
<h1>
  <span> Simona Marcu</span>
</h1>
```
?

Reply:
Same here